augmentation/data_augmenter.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
```

Comment:
Is there a need for two augmentation scripts (`artificial_images_generator.py` and this one)? Just by skimming the code, the scripts are pretty much the same with minor differences.

Reply:
I haven't refactored there yet. Noted that they're pretty much the same.